BUILD.bazel

```diff
@@ -29,24 +29,33 @@ alias(
     visibility = ["//visibility:public"],
 )
 
+exports_files(["LICENSE"])
+
 gazelle(
     name = "gazelle",
     prefix = "github.com/bazelbuild/rules_typescript",
 )
 
-# The node_modules directory is created by `yarn install`
-# WORKAROUND for https://github.com/bazelbuild/bazel/issues/374#issuecomment-296217940
 filegroup(
     name = "node_modules",
-    # Only include files needed for type-checking and runtime
-    srcs = glob([
-        "node_modules/**/*.js",
-        "node_modules/**/*.d.ts",
-        "node_modules/**/*.json",
-    ] + [
-        # Needed because http-server has a bin with no .js extension
-        "node_modules/http-server/**",
-    ]),
+    srcs = glob(
```

Comment:
when will we switch to bazel-managed deps??

Reply:
done